src/libraries/System.ComponentModel.TypeConverter/tests/ILLink.Descriptor.xml

```diff
@@ -0,0 +1,21 @@
+<linker>
+  <assembly fullname="TestUtilities">
+   	<namespace fullname="System" />
+        <type fullname="System.PlatformDetection" />
+  </assembly>
+  <assembly fullname="System" />
+  <assembly fullname="xunit.runner.utility.netcoreapp10" />
+  <assembly fullname="Microsoft.DotNet.XUnitExtensions"/>
+  <assembly fullname="mscorlib" />
```

Comment:
I'm curious if multiple test suites also require assemblies such as `mscorlib` and perhaps `System.PlatformDetection`, would we want to have a common `ILLink.Descriptor.xml` file the test suites can leverage. I guess that will depend on how many test suites end up needing these `ILLink.Descriptor.xml` files.

Reply:
The follow up PR will have many more descriptor files, specific to projects. We can make these changes there. Eg. https://github.com/dotnet/runtime/pull/50885/files#diff-d74ed78dff2160a861191b360b9a3a0b91a0fed05a1109728cc5472801a28799

The process I'm following is to add for specific project first, where it shows up. And then if it shows up in some other projects, then separate that out into a `eng/tests/ildescriptors/` directory, and use that in those projects. If any of these show up in *all* the projects, then we can add them to the "global" ones.

Reply:
I think we should figure out what is common as I fix more tests suites, and move them into common files as necessary.

Reply:
Are these facade assemblies necessary with the latest linker bits?  I know changes went in to preserve facade assemblies that are necessary.

cc @vitek-karas @sbomer @tlakollo @mateoatr

Reply:
@eerhardt I know as of Friday these were necessary to make the test build and run. How late are the "latest"?

Reply:
I guess the support was just for `copy` assemblies. https://github.com/mono/linker/pull/1869. The true fix here is: https://github.com/mono/linker/issues/1837.

Reply:
https://github.com/mono/linker/issues/1837 should be fixed by https://github.com/mono/linker/pull/1869/, `copy` or not. If the linker sees a type string that names a forwarder, the forwarder will be kept (and if not it's a bug).

Reply:
> I guess the support was just for copy assemblies.

Yes. The recent changes keep facade assemblies whenever these are referenced from copy assemblies.

Reply:
To clarify: that change both:
- keeps facades referenced from metadata in `copy` assemblies (like `typeof(Foo)`, etc.)
- keeps facades referenced by string in _any_ marked code path, `copy` or not

Reply:
@naricc - The absolute latest linker was just merged into main - #51162. Can you try your tests without preserving `System` and `mscorlib` again in these files? If you still need it, we should log a bug so we can fix the linker so this "just works".

Reply:
I will try that before merging.

Reply:
@eerhardt It seems they are no longer needed, after rebasing on master, so I have removed them.